svc/ConfigService.js

```diff
@@ -51,5 +60,20 @@ export class ConfigService {
         throwIf(defaultValue === undefined, `Config key not found: '${key}'`);
         return defaultValue;
     }
+    //-----------------
+    // IMPLEMENTATION
+    //-----------------
+
+    createTimer() {
+        Timer.create({
+            runFn: this.checkConfigUpdates,
+            interval: this.REFRESH_INTERVAL * SECONDS
+        });
+    }
+
+    checkConfigUpdates = async () => {
+        const data = await XH.fetchJson({url: 'xh/getConfig'});
+        if (!isEqual(data, this._data)) this._data = data;
+    }
 
```

Comment:
OK -- Actually having auto-reloading configs on the client is something we are very much thinking of supporting in hoist central, but it was not my intention to have as part of this "Timer-Only" change.  Would like to shelve the change in this file for now.

Reply:
And in fact, what's most likely for a client -side timer is going to be responding to a pref.  

This all relates to some related changes Paul just did in Hoist Central -- To make this properly flexible, we should take a "function/closure" as an option to get the interval dynamically!

Reply:
This looks very promising, Brendan -- let's discuss in person -- there a couple of bits I think we need to get, especially handling of calls that take longer than the interval, and avoiding piling them up.

Reply:
I see - so we won't be able to fully replicate the server side functionality until configs are auto-reloaded. But for now, a timer should be able to take three arguments for its `interval`:

1. Integer (current)
2. String (checks for config, purely a convenience at this point)
3. Closure, which could be used to get either a pref or config + add custom logic